src/stories.js

```diff
@@ -90,6 +145,7 @@ export const main = () => {
             />
           </>
         )}
+        merchandise={merchandiseItems?.map((item) => getOption(item))}
```

Comment:
can't we somehow fetch merchandise items from API like we are doing with users?

I think it makes the solution more flexible because now we need to fetch items to every MegaFlow even if most of the workflows can be not related to merch at all...